php/EE/Runner.php

```diff
@@ -55,6 +55,24 @@ private function init_ee4() {
 			mkdir( EE_CONF_ROOT );
 		}
 
+		if ( ! is_dir( EE_CONF_ROOT . '/traefik' ) ) {
+			mkdir( EE_CONF_ROOT . '/traefik' );
+			touch( EE_CONF_ROOT . '/traefik/acme.json' );
+			chmod( EE_CONF_ROOT . '/traefik/acme.json', 600 );
+			file_put_contents(
+				EE_CONF_ROOT . '/traefik/traefik.toml', '################################################################
```

Comment:
```
defaultEntryPoints = ["http", "https"]

InsecureSkipVerify = true

logLevel = "DEBUG"

[entryPoints]
[entryPoints.traefik]
address = ":8080"
 [entryPoints.traefik.auth.basic]
   users = ["easyengine:$apr1$CSR8Nxt6$h/Mid6X/vb6ozs4lrXrcw1"]
[entryPoints.http]
address = ":80"
 [entryPoints.http.redirect]
 entryPoint = "https"
[entryPoints.https]
address = ":443"

[api]
entryPoint = "traefik"
dashboard = true
debug = true

[docker]
domain = "docker.local"
watch = true
exposedByDefault = false

[file]
directory = "/etc/traefik/endpoints/"
watch = true
```

We need this configuration. Please use a library like https://github.com/yosymfony/toml